yet-another-docker-plugin/src/main/resources/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher/init.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
Hmmm, when I first made this change, I thought that Alpine/BusyBox had `env` in `/bin` (and not in `/usr/bin`), but I see that is not true, so I will restore the `/usr/bin/env` reference (although I think `/bin/sh` has a longer history than `/usr/bin/env`).